Source/Atomic/CMakeLists.txt

```diff
@@ -55,10 +55,10 @@ set (SOURCE_FILES ${CONTAINER_SOURCE} ${CORE_SOURCE} ${ENGINE_SOURCE} ${INPUT_SO
                   ${ATOMIC2D_SOURCE} ${ENVIRONMENT_SOURCE}
                   ${SCENE_SOURCE} ${UI_SOURCE} ${SYSTEM_UI_SOURCE}
                   ${WEB_SOURCE} ${SCRIPT_SOURCE} ${METRICS_SOURCE}
-                  ${PLATFORM_SOURCE} ${DATABASE_SOURCE} ${IK_SOURCE})
+                  ${PLATFORM_SOURCE} ${DATABASE_SOURCE} ${IK_SOURCE} ${NAVIGATION_SOURCE})
 
```

Comment:
The Navigation subsystem was originally left out of the WebGL build to reduce the size of generated JS, with WASM around the corner, think we can default this on now